addons/l10n_ro_efactura/models/res_company.py

```diff
@@ -33,17 +33,27 @@ def _compute_l10n_ro_edi_callback_url(self):
             else:
                 company.l10n_ro_edi_callback_url = False
 
+    def _l10n_ro_edi_log_message(self, message: str):
+        self.env['ir.logging'].create({
+            'name': 'l10n_ro_edi_log',
+            'type': 'server',
+            'level': 'INFO',
+            'dbname': self.env.cr.dbname,
+            'message': message,
+            'func': '',
+            'path': '',
+            'line': '',
+        })
+        self.env.cr.commit()
```

Comment:
Need sudo(), and also it might be nice to add the func name so that the logger has more detail on the whereabouts of the error's origin
```suggestion
    def _l10n_ro_edi_log_message(self, message: str, func):
        self.env['ir.logging'].sudo().create({
            'name': 'l10n_ro_edi_log',
            'type': 'server',
            'level': 'INFO',
            'dbname': self.env.cr.dbname,
            'message': message,
            'func': func,
            'path': '',
            'line': 1,
        })
```